onnxruntime/python/tools/tensorrt/perf/build/Dockerfile.custom-trt-perf

```diff
@@ -0,0 +1,64 @@
+# --------------------------------------------------------------
```

Comment:
A few options. This is from https://github.com/nvidia/TensorRT. 
1. Checkout release from the Tensorrt repo + build ORT on top of it. ** can specify cuda version but doesn't have all combinations (how mem leak is using it right now) 
2. Add ubuntu dockerfiles to pipeline folder
3. Use manylinux (had trouble initially)